README.md

```diff
@@ -23,34 +19,6 @@ CDK CLI is a tool to make it easier to get started with [CDK](https://github.com
 We plan to deprecate this CLI and migrate all useful features to the [main Guardian CDK repository](https://github.com/guardian/cdk).
 As a short-term fix for those looking to get a GuCDK stack up and running, here are some manual steps, expectations and notes regarding the process:
 
-### Manual Setup Steps/Expectations
-
-* Ensure your repository has a git remote set (eg `git remote add`) to avoid [`repository_url` errors](https://github.com/guardian/cdk/blob/eed6bfab79d62406b7ee08eee56eddbdf41d6987/src/constructs/core/stack.ts#L149)
-* After generating your scaffolding with `cdk-cli new --init [...args]`, navigate to `/bin/<APP>.ts` and add the stack name to the app declaration
-* **When migrating an application**, make sure you manually remove the `Stage` parameter generated by this CLI. You'll see an error if you don't as this repository currently generates one for you, and this is no longer required as `@guardian/cdk` now handles that behaviour.
-
-### Improvements / TODO
-
-* The `new` command asks for an `APP` and a `STACK`, which corresponds to the `/bin/` filename and the name of the class declared in `/lib`.
-  I think it would be better if the `APP` arg resulted in the name of the app in `/lib`, as, at least for the Guardian, the stack name means both "what stack does this belong to" and "what Cloudformation stack is this",
-  with the latter being more synonymous with the app name.
-
-
-## How to migrate from Cloudformation YAML/JSON
```

Comment:
We have changed our migration strategy since this was written, so we can remove this section and the `migrate` command.